ert3_examples/polynomial/poly.py

```diff
@@ -32,16 +46,17 @@ def valid_coefficients(file_path):
     return arg_parser
 
 
-def _evaluate_polynomial(coefficients):
+def _evaluate_polynomial(coefficients, x_uncertainties):
     a, b, c = coefficients["a"], coefficients["b"], coefficients["c"]
     x_range = tuple(range(10))
-    return tuple(a * x ** 2 + b * x + c for x in x_range)
+    xs = map(sum, zip(x_range, x_uncertainties)) if x_uncertainties else x_range
```

Comment:
What if the `x_uncertainties` are not the same length as `x_range`

Reply:
Probably wise to be strict about it requiring them to be the same length. `zip` will just silently merge until one of the lists is exhausted, which is probably not what we want...